### PR TITLE
enhanced pricing page with available modules clarification

### PR DESCRIPTION
**Pull request by wiswedel**
There's been some confusion among customers/prospects about the available modules of Nextcloud Hub. Let's clarify that on the pricing page.

Question: Shall we call it "Modules"? We haven't been using that term so far but I think it fits best.

Signed-off-by: Sascha Wiswedel <sascha.wiswedel@nextcloud.com>

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@wiswedel huh, is this correct - are Groupware and Talk included with Standard and Premium?

**Comment by wiswedel** [comment]
> @wiswedel huh, is this correct - are Groupware and Talk included with Standard and Premium?

Nono, they're just _available_ for booking.

**Comment by jospoortvliet** [comment]
Ah, I understand. Hmmm. I think readers will be as confused as I am. We could put it in the 'optional components' section, but then people might only want Talk or Groupware (though that is rare).

I think we might need a different approach for this... In the end, this page gives the pricing of Files, not Hub. I don't know a way out, but I don't think that THIS is a good solution.

We could, for now, add Talk and Groupware in the optional components section. Because modules -components - apps, we're just confusing the heck out of everyone. Perhaps we can discuss this at... a contributor week or something...

File: page-pricing.php [hunk 1]
```diff
@@ -54,6 +54,13 @@
...
 					<li class="check" title="<?php echo $l->t('The AGPL license provides legal certainty for all customers and users.');?>" rel="tooltip"><?php echo $l->t('Fully Open Source');?></li>
 				</div>
+				<h1 class="header">Nextcloud modules available</h1>
+				<div class="line"></div>
+				<div class="list">
+					<li class="check" title="<?php echo $l->t('Makes it easy to sync, share and collaborate on your files');?>" rel="tooltip"><?php echo $l->t('<a class="hyperlink" href="https://nextcloud.com/files/">Nextcloud Files</a>');?></li>
+					<li class="nocheck" title="<?php echo $l->t('Secure online meetings, chat and web conferencing');?>" rel="tooltip"><?php echo $l->t('<a class="hyperlink" href="#featureblock-talk">Nextcloud Talk</a>');?></li>
+					<li class="nocheck" title="<?php echo $l->t('Team planning and email made easy');?>" rel="tooltip"><?php echo $l->t('<a class="hyperlink" href="#featureblock-groupware">Nextcloud Groupware</a>');?></li>
```
Comment (gary-kim) [inline page-pricing.php:62]:
```suggestion
					<li class="check" title="<?php echo $l->t('Makes it easy to sync, share and collaborate on your files');?>" rel="tooltip"><a class="hyperlink" href="https://nextcloud.com/files/"><?php echo $l->t('Nextcloud Files');?></a></li>
					<li class="nocheck" title="<?php echo $l->t('Secure online meetings, chat and web conferencing');?>" rel="tooltip"><a class="hyperlink" href="#featureblock-talk"><?php echo $l->t('Nextcloud Talk');?></a></li>
					<li class="nocheck" title="<?php echo $l->t('Team planning and email made easy');?>" rel="tooltip"><a class="hyperlink" href="#featureblock-groupware"><?php echo $l->t('Nextcloud Groupware');?></a></li>
```

File: page-pricing.php [hunk 2]
```diff
@@ -105,6 +112,13 @@
...
 					<li class="check" title="<?php echo $l->t('The AGPL license provides legal certainty for all customers and users.');?>" rel="tooltip"><?php echo $l->t('Fully Open Source');?></li>
 				</div>
+				<h1 class="header">Nextcloud modules available</h1>
+				<div class="line"></div>
+				<div class="list">
+					<li class="check" title="<?php echo $l->t('Makes it easy to sync, share and collaborate on your files');?>" rel="tooltip"><?php echo $l->t('<a class="hyperlink" href="https://nextcloud.com/files/">Nextcloud Files</a>');?></li>
+					<li class="check" title="<?php echo $l->t('Secure online meetings, chat and web conferencing');?>" rel="tooltip"><?php echo $l->t('<a class="hyperlink" href="#featureblock-talk">Nextcloud Talk</a>');?></li>
+					<li class="check" title="<?php echo $l->t('Team planning and email made easy');?>" rel="tooltip"><?php echo $l->t('<a class="hyperlink" href="#featureblock-groupware">Nextcloud Groupware</a>');?></li>
```
Comment (gary-kim) [inline page-pricing.php:120]:
```suggestion
					<li class="check" title="<?php echo $l->t('Makes it easy to sync, share and collaborate on your files');?>" rel="tooltip"><a class="hyperlink" href="https://nextcloud.com/files/"><?php echo $l->t('Nextcloud Files');?></a></li>
					<li class="check" title="<?php echo $l->t('Secure online meetings, chat and web conferencing');?>" rel="tooltip"><a class="hyperlink" href="#featureblock-talk"><?php echo $l->t('Nextcloud Talk');?></a></li>
					<li class="check" title="<?php echo $l->t('Team planning and email made easy');?>" rel="tooltip"><a class="hyperlink" href="#featureblock-groupware"><?php echo $l->t('Nextcloud Groupware');?></a></li>
```

File: page-pricing.php [hunk 3]
```diff
@@ -156,6 +170,13 @@
...
```
Comment (gary-kim) [inline page-pricing.php:178]:
```suggestion
					<li class="check" title="<?php echo $l->t('Makes it easy to sync, share and collaborate on your files');?>" rel="tooltip"><a class="hyperlink" href="https://nextcloud.com/files/"><?php echo $l->t('Nextcloud Files');?></a></li>
					<li class="check" title="<?php echo $l->t('Secure online meetings, chat and web conferencing');?>" rel="tooltip"><a class="hyperlink" href="#featureblock-talk"><?php echo $l->t('Nextcloud Talk');?></a></li>
					<li class="check" title="<?php echo $l->t('Team planning and email made easy');?>" rel="tooltip"><a class="hyperlink" href="#featureblock-groupware"><?php echo $l->t('Nextcloud Groupware');?></a></li>
```

**Comment by jospoortvliet** [comment]
Thanks @gary-kim it's a pita to do the translations in the right way - I want to give translators context (as in, whole sentences) but not too much html - it's a hard balance and there are plenty of places on the site where I got it horribly wrong ;-)

**Comment by jospoortvliet** [comment]
So let's do it this way for now... At some point we need to re-think our pricing page, the design is a bit outdated, too.

**Pull request merged**